swagger.yml

```diff
@@ -1313,6 +1318,9 @@ components:
         code:
           type: string
           description: Card code
+        archived:
```

Comment:
I'd make this required. Card codes can either be active (not archived) or archived. This change requires DB migration that marks all current card codes as active (not archived).